pubspec.yaml

```diff
@@ -2,10 +2,10 @@ name: dart_apitool
 description: A tool to analyze the public API of a package, create a model of it and diff it against another version to check semver.
 repository: https://github.com/bmw-tech/dart_apitool
 
-version: 0.15.1-dev
```

Comment:
The "dev" postfix is kept till the next release happens. The release process will remove it and jump to the next version.